web_widget_radio_tree/static/src/js/widget.js

```diff
@@ -20,22 +20,16 @@ openerp.web_widget_radio_tree = function (instance) {
     instance.web.form.RadioTree = instance.web.form.FieldBoolean.extend({
         template: 'RadioTree',
         start: function() {
-            var self = this;
+            this._super.apply(this, arguments);
```

Comment:
it's really crucial to return super's result, because this is usually a deferred object and it can mess up stuff seriously to just drop it: https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L653